### DISCUSSION TITLE
Spotlights and Cameras should show in Ship Status Monitor (same as Guns)

### DISCUSSION THREAD

**Discussion by Saevon** [FakeFishGames/Barotrauma#13119] [Ideas and suggestions]
When looking at the status monitor it shows you the locations of all the Turrets, however there is no such helpful icon for spotlights or cameras!

**Comment by Ek-Videogames** [comment]
A toggleable property would be good, so as designs using decorative spotlights (as there is no other non-turreted directional light in vanilla) wouldn't have status monitor display cluttered with them.
